src/components/ItaliaTheme/Blocks/Listing/Commons/ListingImage.jsx

```diff
@@ -20,7 +20,11 @@ const ListingImage = ({
     return (
       <Image
         className={className}
-        image={item[item.image_field] || item.image || item['@id']}
+        image={
+          item.image_scales?.[item.image_field]?.[0] ||
+          item.image ||
+          item['@id']
```

Comment:
Forse manca `item[item.image_field]` come caso aggiuntivo dopo il primo nuovo che hai inserito? Lo sto dicendo "a occhio", devo ancora testare.

Reply:
In realtà no. anche `image` alla fine è un field che nel listingimage (che ha solo i metadata del catalogo) non avrebbe neanche di base, tantomeno avrebbe un field di nome `item.image_field`